.github/workflows/vib-publish.yaml

```diff
@@ -0,0 +1,113 @@
+name: 'VIB Publish'
+on: # rebuild any PRs and main branch changes
+  push:
+    branches:
+      - master
+    paths:
+      - 'bitnami/wordpress/Chart.yaml'
+env:
+  CSP_API_URL: https://console.cloud.vmware.com
+  CSP_API_TOKEN: ${{ secrets.CSP_API_TOKEN }}
+  VIB_PUBLIC_URL: https://cp.bromelia.vmware.com
+jobs:
+  get-chart:
```

Comment:
Do we really need this job for those changes landing in `stable` branch? I think we can guarantee that only one Helm Chart has been modified. Am I wrong?